packages/salesforce-adapter/src/client/user_facing_errors.ts

```diff
@@ -15,36 +15,57 @@
 */
 
 import { DeployMessage } from 'jsforce'
+import { decorators, values } from '@salto-io/lowerdash'
+import _ from 'lodash'
+import { logger } from '@salto-io/logging'
+import { ENOTFOUND, ERROR_HTTP_502, INVALID_GRANT, SF_REQUEST_LIMIT_EXCEEDED } from '../constants'
 
-const ERROR_HTTP_502 = 'ERROR_HTTP_502'
-const REQUEST_LIMIT_EXCEEDED = 'sf:REQUEST_LIMIT_EXCEEDED'
-const INVALID_GRANT = 'invalid_grant'
-const ENOTFOUND = 'ENOTFOUND'
+const log = logger(module)
+const { isDefined } = values
 
 
 /**
  * To allow more robust support and avoid boilerplate
  * for supporting different Error objects (e.g. Node DNSError, JSForce error)
  * this values can be any of the Error object properties.
  */
-const MAPPABLE_ERROR_PROPERTIES = [
+export const MAPPABLE_ERROR_PROPERTIES = [
   ERROR_HTTP_502,
-  REQUEST_LIMIT_EXCEEDED,
+  SF_REQUEST_LIMIT_EXCEEDED,
   INVALID_GRANT,
   ENOTFOUND,
 ] as const
 
 export type MappableErrorProperty = typeof MAPPABLE_ERROR_PROPERTIES[number]
```

Comment:
Nit: name: is the `property` text here meaningful ? can we remove it please?

```suggestion
export type MappableError = typeof MAPPABLE_ERROR_PROPERTIES[number]
```